### DISCUSSION TITLE
show / hide menu access

### DISCUSSION THREAD

**Discussion by juangdnar** [noxoua/filament-activity-log#50] [Q&A]
How can I show or hide the access from the main menu, from resources that I have installed through the plugins, for example I am using the activity log plugin and I don't know how to condition it so that it is shown only to administrator-type users. annex example image; Thanks in advance.

![image](https://github.com/noxoua/filament-activity-log/assets/115037170/2404e2dd-94a9-4991-9467-3c8a1bd51e45)

**Comment by noxoua** [comment]
My apologies for the late response. For your requirement, you can use the `shouldRegisterNavigation` method as detailed in the [Filament Documentation](https://filamentphp.com/docs/3.x/panels/pages#conditionally-hiding-pages-in-navigation).
